tests/integ/jobmanager/common.go

```diff
@@ -394,8 +394,8 @@ func (suite *TestJobManagerSuite) TestJobManagerJobCrash() {
 	// the report calculate by the plugin will be nil
 	ev, err := pollForEvent(suite.eventManager, jobmanager.EventJobFailed, types.JobID(jobID))
 	require.NoError(suite.T(), err)
-	require.Equal(suite.T(), 1, len(ev))
-	require.Equal(suite.T(), string(*ev[0].Payload), "{\"Err\":\"TestStep crashed\"}")
+	require.Equal(suite.T(), 2, len(ev))
+	require.Equal(suite.T(), "{\"Err\":\"error at test step 'Crash' (label: 'crash_label'): TestStep crashed\"}", string(*ev[0].Payload))
```

Comment:
updated error string, and inverted the order, since `require.Equal` takes `(error, expected, actual)`, not `(error, actual, expected)`